Reddit/Reddit/Servises/PostService.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Reddit.DBServis;
+using Reddit.Entities;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Reddit.Servises
+{
+    public class PostService : IPostService
+    {
+        private readonly RedditDbContext db;
+
+        public PostService(RedditDbContext db)
+        {
+            this.db = db;
+        }
+        public void CreatePost(Post post)
+        {
+            db.Add(post);
+            db.SaveChanges();
+        }
+
+        public void DeletePost(int id)
+        {
+            var post = GetPost(id);
+            db.Posts.Remove(post);
+            db.SaveChanges();
+        }
+
+        public IEnumerable<Post> FindPost(string searching)
+        {
+            return db.Posts.Where(p => p.Titel.Contains(searching) || p.PostUrl.Contains(searching));
+        }
+
+        public IEnumerable<Post> Gell10BestPost()
```

Comment:
Overall using number in names is not the best. But maybe you can use more generic appropach.
Like GetTopPost(int count) But its a just suggestion